Xamarin.Forms.Platform.Android/FastRenderers/MediaElementRenderer.cs

```diff
@@ -0,0 +1,480 @@
+﻿using Android.Content;
+using Android.Media;
+using Android.Views;
+using Android.Widget;
+using System;
+using System.ComponentModel;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform.Android.FastRenderers;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public sealed class MediaElementRenderer : FrameLayout, IVisualElementRenderer, IViewRenderer, IEffectControlProvider, MediaPlayer.IOnCompletionListener, MediaPlayer.IOnInfoListener, MediaPlayer.IOnPreparedListener, MediaPlayer.IOnErrorListener
```

Comment:
@samhouts do we need this to be sealed? I realize it's experimental but seeing as how it's experimental being able to extend the renderer seems even more important because of the possibility of needing to add behavior or work around bugs